docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -175,6 +175,10 @@ Use the same terminal window as the previous step.
      Processing config-genesis.batch...
      Generating /var/lib/sawtooth/genesis.batch
 
+   .. note::
+
+      The `-u sawtooth` refers to the sawtooth user, not the sawtooth command.
```

Comment:
Please use double-backticks around ``-u sawtooth``.  Also (as long as you're changing this line), I'd say "The ``-u sawtooth`` option.